src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer.Tests/Personalizer.Tests.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>Microsoft.Azure.CognitiveServices.Personalizer.Tests Class Library</Description>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Personalizer.Tests</AssemblyName>
-    <Version>1.0.0</Version>
+    <Version>0.8.0-preview</Version>
```

Comment:
Test project version must be set to 1.0.0 and never modified. Not blocking but must be fixed

Reply:
fixed